types/office-js-preview/index.d.ts

```diff
@@ -12214,7 +12214,7 @@ declare namespace Office {
          */
         getInitializationContextAsync(callback: (asyncResult: Office.AsyncResult<string>) => void): void;
         /**
-         * Returns string values in the selected item that match the regular expressions defined in an XML manifest file.
+         * Returns string values in the selected item that match the regular expressions defined in an add-in only manifest file.
          *
          * @returns
          * An object that contains arrays of strings that match the regular expressions defined in the manifest XML file.
```

Comment:
Confirming that we're keeping "XML" instances like this?

```suggestion
         * An object that contains arrays of strings that match the regular expressions defined in the add-in manifest file.
```